src/content/reference/rules/never-pass-around-hooks-as-regular-values.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Never pass around hooks as regular values
+---
+
+<Intro>
+Hooks should only be called inside of components. Never pass it around as a regular value.
+</Intro>
+
+---
+
+Hooks allow you to augment a component with React features. They should always be called as a function, and never passed around as a regular value. This enables _local reasoning_, or the ability for developers to understand everything a component can do by looking at that component in isolation.
```

Comment:
> enables local reasoning

This makes it sound like this is less of a rule and more of a recommended pattern. It's unclear what the consequence of the bad example below is.